packages/tabs/src/TabsContainer.tsx

```diff
@@ -23,10 +23,11 @@ TabsContainer.defaultProps = {
 TabsContainer.propTypes = {
   children: PropTypes.func,
   render: PropTypes.func,
+  tabs: PropTypes.arrayOf(PropTypes.any).isRequired,
```

Comment:
Some typescript weirdness here. I can't specify `PropTypes.shape({ value: PropTypes.any, disabled: PropTypes.bool })` within `arrayOf`, for some reason.

Reply:
I experienced the same with `ComboboxContainer` `options`.